### PR TITLE
Multiple fixes to the attribute editor

### PR DESCRIPTION
**Pull request by dnlkoch**
This fixes the attribute editor of the `draw` tools: 

- Allow adding multiple rows at once (without the need to save the feature first).
- Don't delete the attribute directly on the feature.
- Remove unneeded state copy of the feature attributes
- Show all available attributes in a dropdown

### CODE REVIEW

**Comment by faouziH21** [comment]
User feedback is very important for non-devs; hence the changes made in [f1df09e](https://github.com/terrestris/shogun-gis-client/pull/1506/commits/f1df09eb624c4ec78bb9a5252fc7fffbfaf1e35e) .
This PR focuses on the "Edit" function. It adds feedback for saving (pressing the "Apply" button), a Popup confirmation for deleting attributes and a heads up for saving changes.

File: src/components/ToolMenu/Draw/Attributions/index.tsx [hunk 1]
```diff
@@ -1,67 +1,134 @@
...
+        if (!(feat.get(prop) instanceof OlGeometry)) {
+          featureCollectionAttributes.add(prop);
+        }
+      });
     });
-  }, [selectedFeature, form]);
+
+    setAvailableFeatureCollectionAttributes(Array.from(featureCollectionAttributes));
+
+    // updateAvailableFeatureAttributes(form.getFieldsValue());
```
Comment (dnlkoch) [inline src/components/ToolMenu/Draw/Attributions/index.tsx:126]: Line can be removed.

File: src/components/ToolMenu/Draw/Attributions/index.tsx [hunk 2]
```diff
@@ -123,86 +185,129 @@ const AttributionDrawer: React.FC<AttributionDrawerProps> = ({
...
+            title={t('AttributionPopConfirm.title')}
+            okText={t('AttributionPopConfirm.okText')}
+            cancelText={t('AttributionPopConfirm.cancelText')}
+            placement='topRight'
+            onConfirm={removeAndMessage}
+          >
+            {contextHolder}
+            <Button
+              className="remove-attribute-button"
+              // onClick={console.log('hi')}
```
Comment (dnlkoch) [inline src/components/ToolMenu/Draw/Attributions/index.tsx:225]: Line can be removed.

File: src/components/ToolMenu/Draw/Attributions/index.tsx [hunk 3]
```diff
@@ -123,86 +185,129 @@ const AttributionDrawer: React.FC<AttributionDrawerProps> = ({
...
-        className='attribution-drawer'
-        placement="right"
-        mask={false}
-        maskClosable={false}
-        onClose={handleClose}
-        {...passThroughProps}
-      >
+    <Drawer
+      title={t('Attribution.title')}
+      className='attribution-drawer'
```
Comment (dnlkoch) [inline src/components/ToolMenu/Draw/Attributions/index.tsx:243]:
```suggestion
      className="attribution-drawer"
```

File: src/i18n/translations.ts [hunk 4]
```diff
@@ -233,9 +233,21 @@ export default {
...
+        title: 'Attribut löschen?',
+        okText: 'Löschen',
+        cancelText: 'Abbrechen'
+      },
       ResetButton: {
         title: 'Zurücksetzen'
       },
+      notificationDeleteText:{
+        title: 'Drücke "Speichern" um die Änderung zu bestätigen.',
+        info: 'Wenn auf „Übernehmen“ nicht geklick wird, werden alle an dieser Zeichnung vorgenommenen Änderungen verworfen.'
```
Comment (dnlkoch) [inline src/i18n/translations.ts:246]:
```suggestion
        info: 'Wenn "Übernehmen" nicht geklickt wird, werden alle an dieser Zeichnung vorgenommenen Änderungen verworfen.'
```

File: src/components/ToolMenu/Draw/Attributions/index.tsx (hunk 1 above)
Comment (faouziH21) [reply src/components/ToolMenu/Draw/Attributions/index.tsx]: done in: [1410fd6](https://github.com/terrestris/shogun-gis-client/pull/1506/commits/1410fd6bc66d3716fe9b90f51323beea1a3978ef)

File: src/components/ToolMenu/Draw/Attributions/index.tsx (hunk 3 above)
Comment (faouziH21) [reply src/components/ToolMenu/Draw/Attributions/index.tsx]: done in: [1410fd6](https://github.com/terrestris/shogun-gis-client/pull/1506/commits/1410fd6bc66d3716fe9b90f51323beea1a3978ef)

**Review by ahennr — COMMENTED** [review]
> Show all available attributes in a dropdown (WIP)

Still WIP?

**Comment by dnlkoch** [comment]
Ah, good catch @ahennr. Fixed the description since it's implemented now.

**Review by ahennr — APPROVED** [review]

**Pull request merged**